### DISCUSSION TITLE
/latest API?

### DISCUSSION THREAD

**Discussion by farooqkz** [posativ/isso#772] [Q&A]
Hello.
I cannot find out where can I access `/latest` API(https://github.com/posativ/isso/blob/master/isso/views/comments.py#L1197). Trying `isso_root/latest?limit=10` but getting 404 instead of JSON.
Using latest release from PyPI.
Any ideas?

**Comment by jelmer** [comment]
Do you have the "latest-enabled" option enabled?

**Comment by farooqkz** [comment]
Nope I have not and now I see why it was throwing back 404 to me...

Actually I was looking for an API just for admin with which admin, and just admin, could iterate over comments and do something like verify or delete on all comments matching a filter. This way any filter matching will be done client side with JS rather than server side.

**Comment by farooqkz** [comment]
By the way, I saw that Github has marked me as contributor for this repo and I didn't remember what contributions I have made till I saw my PR :D

**Comment by ix5** [comment]
@farooqkz would you mind closing this?

**Comment by farooqkz** [comment]
You mean I close this one and open a separate issue for the question which I asked?

**Comment by ix5** [comment]
To answer your other question, afaik there is no explicit admin API, but rather admin uses the standard endpoints that the client also uses (`/id/<action>`), just using an admin cookie and a comment hash instead.

See e.g.
https://github.com/posativ/isso/blob/9e021cb6263e921ac519f52b82df3f66a297d3d1/isso/js/admin.js#L27-L33

So you can get a list of all comments via the `latest` API if you so desire and then filter them, get yourself an admin cookie and then delete/approve/whatever the respective comments. But why go to all that trouble when you can just edit the sqlite database directly?
